src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -226,10 +226,19 @@ public void PushWithTransition(Page page)
             transitionSet = CreateTransitions(topPage, page, true);
             transitionSet.Finished += (object sender, EventArgs e) =>
             {
+                HideContentOfPage(topPage);
+
                 if (page is DialogPage == false)
                 {
                    topPage.SetVisible(false);
                 }
+                else
+                {
+                    topPage.SetVisible(true);
```

Comment:
Is this code required here?

Reply:
No, it can be removed now, too.
Thank you for reviewing it.